python/mxnet/gluon/model_zoo/vision/resnet.py

```diff
@@ -102,10 +102,10 @@ class BottleneckV1(HybridBlock):
     def __init__(self, channels, stride, downsample=False, in_channels=0, **kwargs):
         super(BottleneckV1, self).__init__(**kwargs)
         self.body = nn.HybridSequential(prefix='')
-        self.body.add(nn.Conv2D(channels//4, kernel_size=1, strides=1))
+        self.body.add(nn.Conv2D(channels//4, kernel_size=1, strides=stride))
```

Comment:
1. How about add 'use_bias=False' in Conv2D? This will make resnet faster.
   The effect of bias in convolution is same as beta in the next batchnorm.
2. Does (2,2) stride set in 1*1 conv or 3*3 conv will effect the final accuracy of resnet? In 1*1 conv, (2,2) stride will loss some pixels in feature map.

Reply:
In my implementation, using stride=2 on 1x1 conv has about 0.3 lower top-1 accuracy.